src/share/poudriere/jail.sh

```diff
@@ -355,6 +379,10 @@ update_jail() {
 		err 1 "Unsupported method"
 		;;
 	esac
+	pkgbase=$(jget ${JAILNAME} pkgbase)
+	if [ ! -z "${pkgbase}" ] && ${pkgbase} -eq 1 ]; then
```

Comment:
There's a syntax error here. Missing `[` after `&&`.
Also should quote the `${pkgbase}`
And `-n` rather than `! -z` is more the style used.
And need a `local pkgbase` at the top.